pkg/apis/integreatly/v1alpha1/rhmiconfig_types.go

```diff
@@ -143,3 +148,86 @@ func (c *RHMIConfig) ValidateDelete() error {
 func init() {
 	SchemeBuilder.Register(&RHMIConfig{}, &RHMIConfigList{})
 }
+
+// ValidateBackupAndMaintenance ensures that the backup and maintenance times
+//   * are not empty
+//   * are correctly formatted
+//   * do not overlap
+// If the times are valid, 1 hour non-overlapping backup and maintenance windows
+// are returned as a result in a format required by AWS
+func ValidateBackupAndMaintenance(backupApplyOn, maintenanceApplyFrom string) (string, string, error) {
```

Comment:
@maleck13 @philbrookes 

Can I get a final confirmation on the approach that will be taken with windows for GA? Will an end user provide a single maintenance window that will contain both the AWS Backup and Maintenance windows, or will the end user provide two windows separately? I had thought we were moving to the first one, the RHMIConfig CR still seems to have them separated out though.